src/ssl.c

```diff
@@ -29369,6 +29369,28 @@ void* wolfSSL_GetHKDFExtractCtx(WOLFSSL* ssl)
     }
 #endif /* OPENSSL_EXTRA || OPENSSL_EXTRA_X509_SMALL */
 
+#if defined(OPENSSL_ALL)
+/* Returns the oid buffer from the short name or long name of an ASN1_object
+ * and NULL on failure */
+const byte* wolfSSL_OBJ_txt2oidBuf(char* buf, word32* inOutSz, word32 oidType)
```

Comment:
difference in indent here. All should be indented or all not indented

Reply:
Noticing that this is a WOLFSSL_API function addition, is there a way we could work around having a new public API? Such as using wolfSSL_OBJ_txt2nid and wolfSSL_OBJ_nid2obj from wolfCLU? Then getting the string from the object struct returned.